src/frontends/onnx/docs/tests.md

```diff
@@ -4,9 +4,126 @@
 - [C++ gtest-based tests](../tests)
 - [Python frontend tests](../../../../src/bindings/python/tests/test_frontend)
 - [Python operators tests](../../../../src/bindings/python/tests/test_onnx)
-- [Python compliance with ONNX standard tests](../../../../src/bindings/python/tests/test_onnx/test_backend.py)
+- [Python tests to confirm operators compliance with the ONNX standard](../../../../src/bindings/python/tests/test_onnx/test_backend.py)
 - [Python OpenModelZoo tests](../../../../src/bindings/python/tests/test_onnx/test_zoo_models.py)
+- [OpenVINO™ Execution Provider for ONNX Runtime tests](../../../../.ci/azure/linux_onnxruntime.yml)
 
 
-## How to run tests
-TBD - 93002
\ No newline at end of file
+## How to run the tests locally
+## C++ tests
+1. Build OpenVINO with `-DENABLE_TESTS=ON` flag.
+2. Run `<OV_REPO_DIR>/bin/intel64/<OV_BUILD_TYPE>/ov_onnx_frontend_tests`, where <OV_REPO_DIR> is your workspace folder, `OV_BUILD_TYPE` can be `Debug` or `Release`. It depends on `-DCMAKE_BUILD_TYPE` cmake build option.
+You can filter tests using `--gtest_filter` flag like any other gtest-based tests.
+
+Example command:
+```
+<OV_REPO_DIR>/bin/intel64/<OV_BUILD_TYPE>/ov_onnx_frontend_tests --gtest_filter=*add*
+
+```
+
+
+## Pre-steps for all Python tests:
+1. Build OpenVINO with `-DENABLE_PYTHON=ON`, preferably in a `Python` virtual environment. In order to avoid problems with many Python interpreters installed on the host, it is worth to set also `-DPYTHON_EXECUTABLE=PYTHON_INTERPRETER_PATH` build option.
+Note: if you want to run the tests from installation directory location (like in the CI), you should add `-P cmake_install.cmake` and `-DCOMPONENT=tests` cmake build options and install OpenVINO (via `cmake --build . --target install`) as additional steps.
+2. Set-up `Python` paths via `source <OV_INSTALL_DIR>/setupvars.sh` (or `sh <INSTALL_DIR>\setupvars.bat` for `Windows`) or set `export PYTHONPATH=<OV_REPO_DIR>/bin/intel64/<BUILD_TYPE>/python_api/python<PYTHON_VERSION>` (note that `set` instead of `export` should be used for `Windows`).
```

Comment:
I'd rather keep the setupvars approach only here. Manual handling of variables might become obsolete at some point and this document should be adjusted in such case.

Reply:
agree, removed